spec/acceptance/email_notification_spec.rb

```diff
@@ -52,6 +54,14 @@ def post_notification
         post_notification
         expect(last_response.body).to eq('')
       end
+
+      describe 'POSTing a Amazon SES Setup Notification to /user-signup/email-notification' do
```

Comment:
Aware this is from legacy, but would rather a whitelist be used as apposed to a black list.
IE, check for the messageId we want, not the one we don't care about